apps/schema/src/components/Schema.vue

```diff
@@ -184,7 +184,7 @@ export default {
       this.loading = true;
       request(
         "graphql",
-        "{_session{schemas,roles}_schema{name,tables{name,tableType,inherit,externalSchema,description,semantics,columns{name,table,position,columnType,inherited,key,refSchema,refTable,refLink,refBack,required,readonly,description,semantics,validation,visible} } } }"
+        "{_session{schemas,roles}_schema{name,tables{name,tableType,inherit,externalSchema,description,semantics,columns{id,name,table,position,columnType,inherited,key,refSchema,refTable,refLink,refBack,required,readonly,description,semantics,validation,visible} } } }"
```

Comment:
Maybe nice to indent or use a external graphql file here

Reply:
You can import { gql } from  "graphql-request"; and use that to make the formatting better. I also really like having some extensions for VS-code that assist with gql highlights and so.